demos/gesture_recognition_demo/cpp_gapi/include/tracker.hpp

```diff
@@ -0,0 +1,292 @@
+// Copyright (C) 2021 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#pragma once
+
+#include <memory>
+#include <set>
+#include <string>
+#include <tuple>
+#include <unordered_map>
+#include <utility>
+#include <vector>
+#include <utils/kuhn_munkres.hpp>
+
+struct TrackedObject {
+    cv::Rect rect = cv::Rect();
```

Comment:
Not sure if it is required, a default constructor should be called automatically?